js/utility/bc-google-analytics.js

```diff
@@ -79,13 +79,47 @@ bcpl.utility.googleAnalytics = (() => {
 		addOutboundLinkTracking();
 	};
 
+	const trackEvent = (action, category, label, value) => {
+		const options = {
+			'event_category' : category, 
+			'event_label' : label
+		}
+
+		if (value) {
+			options['value'] = value;
+		}
+
+		gtag('event', action, options);
+	};
+
+	const trackLogin = () => {
+		gtag('event', 'login', { method: 'Google' });
```

Comment:
instead of using gtag here, i would call trackEvent.

What the docs here are saying is using this default  gtag('event', 'login', then our customer value here);

So for example, login could be 'Polaris'.

You might want to pass "loginType as a parameter"